app/views/pair_requests/_table_row.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class="table-row odd:bg-white even:bg-neutral-100 hover:bg-neutral-200 divide-x-2 divide-black">
   <%= render TableCellComponent.new(container_link: pair_request) do %>
-    <%= pair_request.partner_for(current_user).email %>
+    <% partner = pair_request.partner_for(current_user) %>
+    <p><%= partner.first_name %></p>
+    <p><%= partner.last_name %></p>
   <% end %>
```

Comment:
should we not use `full_name` here?

Reply:
These are intentionally placed in separate block level elements so they are rendered to HTML on separate lines:
![Selection_214](https://github.com/agency-of-learning/PairApp/assets/88392688/0bcc7413-4e32-4e13-b57c-0bbcc84a0ebb)
For the table specifically, I think this looks good, especially on mobile. It also matches the way they're done in the wireframes.